CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Contributing Guidelines
+
+Welcome to Shipwright Build. We are excited about the prospect of you contributing to our project. Your support is more than welcome!
+
+## Getting Started
+
+We have initial documentation on how to start contributing here:
+
+- Learn how to [change shipwright and try out your changes on a local cluster](/docs/development/local_development.md)
+- Our main [documentation](/docs/)
+- Our [Code of Conduct](/code-of-conduct.md)
+
+## Creating new Issues
+
+We recommend to open an issue for the following scenarios:
+
+- Asking for help or questions. (_Use the **discussion** or **help_wanted** label_)
+- Reporting a Bug. (_Use the **bug** label_)
+- Requesting a new Feature. (_Use the **enhancement** label_)
+
+_Note_: Do not hesitate to contact us inmediately in our [Slack channel](https://kubernetes.slack.com/archives/C019ZRGUEJC).
+
+## Writing Pull Requests
+
+For making `noice` in the repository and creating new contributions, here are some recommendations:
+
+- Fork the project
```

Comment:
```suggestion
- Fork the project into your personal Github repository
```

Reply:
Would not it be rather

Fork the project into your personal GitHub **account**

?

Reply:
correct - *account* is probably better phrasing.

Reply:
done